lib/Transforms/Instrumentation/ShadowMemSeaDsa.cc

```diff
@@ -1146,7 +1164,7 @@ CallInst &ShadowDsaImpl::getParentDef(CallInst &memOp) {
 
   auto *fn = memOp.getCalledFunction();
   assert(fn);
-  assert(fn == m_memLoadFn || fn == m_memStoreFn || fn == m_memTrsfrLoadFn);
+  //assert(fn == m_memLoadFn || fn == m_memStoreFn || fn == m_memTrsfrLoadFn);
```

Comment:
I will uncomment it, sorry

Reply:
It would be good to fix the code as well. Please log an issue in `sea-dsa` with code on which the assertion fails. Remember that until it is fixed, you should disable this `sea-dsa` optimization.